public/themes/reborn/templates/dark.css

```diff
@@ -48,12 +52,15 @@ hr {
 
 input[type=password],
 input[type=text],
+input[type=checkbox],
+input[type=radio],
 input[type=url],
 input[type=year],
+select,
 textarea {
-    background-color: #f5f5f5;
-    background-image: linear-gradient(to top, #eee, #fff);
-    border-color: #fff;
+    background-color: #000;
+    background-image: linear-gradient(to top, hsl(0, 0%, 8%), hsl(0, 0%, 4%));
+    border-color: #222;
```

Comment:
should we make this light? / delete it?
![image](https://github.com/ampache/ampache/assets/1305249/70461a09-6f51-4084-aa3a-83298a2e65e1)

Not sure what the fallback colour is but boxes can be a bit hard to see with 222 set

Reply:
Give it

```
background-color: hsl(0, 0%, 10%);
background-image: linear-gradient(to top, hsl(0, 0%, 12%), hsl(0, 0%, 8%));
border-color: hsl(0, 0%, 30%);
```

Reply:
yeah that looks heaps better, i'll add that